src/qt/signverifymessagedialog.cpp

```diff
@@ -89,6 +89,7 @@ void SignVerifyMessageDialog::on_addressBookButton_SM_clicked()
 {
     if (model && model->getAddressTableModel())
     {
+        model->refresh(true);
```

Comment:
`model->refresh(true /* pk_hash_only */);`

Reply:
Implemented with your other suggestion in c4ea501e96363e937200bc97b8e2d78162bdb699